mix.exs

```diff
@@ -58,6 +58,7 @@ defmodule EpochElixir.Mixfile do
       {:sext, [github: "uwiger/sext", tag: "1.4.1", manager: :rebar, override: true]},
       {:edown, "~> 0.8", override: true},
       {:enacl, github: "aeternity/enacl", ref: "2f50ba6", override: true},
+      {:msgpax, "~> 2.1.1"},
```

Comment:
Why was MsgPack added again?

Reply:
Thank you for your review, I will remove it.